packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -382,6 +390,8 @@ export function buildRootObject(_powers, _params, testJigSetter = undefined) {
       getBrandForIssuer: issuer => issuerTable.getByIssuer(issuer).brand,
       getIssuerForBrand: brand => issuerTable.getByBrand(brand).issuer,
       getAmountMath,
+      getDonePromise: () => E(E(zoeInstanceAdmin).getAdminNode()).done(),
+      getAdminData: () => E(E(zoeInstanceAdmin).getAdminNode()).adminData(),
```

Comment:
Can we brainstorm the API here together? It seems like we are just copying the adminNode API, but I don't think that's the right answer. First, tightly coupling the two APIs seems it will hurt us in the future, and second, putting the same methods on ZCF changes the context so that the method names no longer make as much sense. 

What about:
```suggestion
      getVatShutdownPromise: () => E(zoeInstanceAdmin).getVatShutdownPromise(),
      getVatStats: () => E(zoeInstanceAdmin).getVatStats(),
```
`getVatShutdownPromise` is too long, but it's exactly descriptive.

Reply:
The accessors have moved to zoe with your suggested names. 

Brian pointed out that these promises are threaded through the ZCF vat and won't work when the vat shuts down.